--- remote/gfal.py
+++ remote/gfal.py
@@ -150,6 +150,9 @@ def upload(self):
         source = "file://" + os.path.abspath(self.local_file())
         # disable all timeouts (file transfers can take a long time)
         self._gfal("copy", "-p", "-f", "-n", "4", "-t", "0", "-T", "0", source, target)
+        
+    def remove(self):
+        self._gfal("rm", "-r", self.remote_file())
 
     @property
     def list(self):
